optuna/integration/skopt.py

```diff
@@ -0,0 +1,219 @@
+from __future__ import absolute_import
+
+import numpy as np
+
+from optuna import distributions
+from optuna.samplers import BaseSampler
+from optuna.samplers import TPESampler
+from optuna.structs import StudyDirection
+from optuna import types
+
+try:
+    import skopt
+    from skopt.space import space
+
+    _available = True
+except ImportError as e:
+    _import_error = e
+    # SkoptSampler is disabled because Scikit-Optimize is not available.
+    _available = False
+
+if types.TYPE_CHECKING:
+    from typing import Any  # NOQA
+    from typing import Dict  # NOQA
+    from typing import List  # NOQA
+    from typing import Optional  # NOQA
+    from typing import Set  # NOQA
```

Comment:
`Set` is not used in this file. Please remove it.

Reply:
Thank you for your careful review!

Fixed: https://github.com/pfnet/optuna/pull/399/commits/6598c4c99af4244d829a4bbab1e7fe0695306c72